src/libNode/FinalBlockProcessing.cpp

```diff
@@ -1459,6 +1459,7 @@ bool Node::ProcessFinalBlock(const vector<unsigned char>& message,
 
     unsigned int cur_offset = offset;
 
+    // Initialize it with 255
     uint8_t shard_id = (uint8_t)-1;
```

Comment:
Why not use 255 instead?

Reply:
I have no idea...lol

Reply:
Maybe the purpose is to initialize with the largest number in the valid range, despite from the size of the unsigned int